### PR TITLE
Drumstick.dmidiplayer version 1.6.0

### PR DESCRIPTION
**Pull request by pedrolcl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/60988)

### CODE REVIEW

File: manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+FileExtensions:
+- kar
+- mid
+- midi
+- rmi
+- wrk
+ReleaseDate: 2022-05-17
+Installers:
+- Architecture: x64
+  InstallerUrl: http://downloads.sourceforge.net/project/dmidiplayer/v1.6.0/dmidiplayer-1.6.0-win-x64-setup.exe
```
Comment (ItzLevvie) [inline manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.installer.yaml:25]:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/dmidiplayer/files/v1.6.0/dmidiplayer-1.6.0-win-x64-setup.exe/download
```
for consistency with other sourceforge packages.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+PackageIdentifier: Drumstick.dmidiplayer
+PackageVersion: 1.6.0
+PackageLocale: en-US
+Publisher: Drumstick Projects
+PublisherUrl: https://dmidiplayer.sourceforge.io
+PublisherSupportUrl: https://sourceforge.net/p/dmidiplayer/tickets/
+# PrivacyUrl: 
+Author: Pedro López-Cabanillas <plcl@users.sourceforge.net>
+PackageName: Drumstick Multiplatform MIDI File Player
```
Comment (ItzLevvie) [inline manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml:12]:
```suggestion
PackageName: dmidiplayer
```
This must match Control Panel.

File: manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS0.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Drumstick.dmidiplayer
+PackageVersion: 1.6.0
+PackageLocale: en-US
+Publisher: Drumstick Projects
```
Comment (ItzLevvie) [inline manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml:7]:
```suggestion
Publisher: Drumstick
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+PublisherUrl: https://dmidiplayer.sourceforge.io
+PublisherSupportUrl: https://sourceforge.net/p/dmidiplayer/tickets/
+# PrivacyUrl: 
+Author: Pedro López-Cabanillas <plcl@users.sourceforge.net>
+PackageName: Drumstick Multiplatform MIDI File Player
+PackageUrl: https://sourceforge.net/projects/dmidiplayer/
+License: GPLv3
```
Comment (ItzLevvie) [inline manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml:14]:
```suggestion
License: GPL-3.0-only
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml (hunk 2 above)
Comment (pedrolcl) [reply manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml]: OK

**Review by ItzLevvie — APPROVED** [review]

File: manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml (hunk 4 above)
Comment (pedrolcl) [reply manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml]: I've changed the License field to the correct SPDX short identifier, but not the suggested one.

**Pull request merged**